### PR TITLE
Improve A11y in the demo

### PR DESCRIPTION
**Pull request by blasten**
Issue: https://github.com/PolymerElements/iron-dropdown/issues/23 

cc @notwaldorf

### CODE REVIEW

**Comment by cdata** [comment]
What's up with the failing tests? :(

**Comment by cdata** [comment]
(can you rebase this branch?)

**Comment by blasten** [comment]
@cdata it's rebased now. I'm not sure why this test is still failing. They all passed locally.

File: test/iron-dropdown.html [hunk 1]
```diff
@@ -149,8 +149,10 @@
         test('can be disabled with `allowOutsideScroll`', function() {
```
Comment (cdata) [inline test/iron-dropdown.html]: Shouldn't this be an `async` test now? (missing `done` argument)

**Comment by blasten** [comment]
@cdata I dropped the `async` test. These two tests are related to `Polymer.IronDropdownScrollManager`

**Comment by cdata** [comment]
> Probably we can fix them in a separate PR.
Can you point me to the separate PR that will fix these tests when merged?

**Comment by cdata** [comment]
Hrm. It looks like you only changed the demo. Weird.

**Comment by cdata** [comment]
LGTM :+1:

**Pull request merged**